src/python/bot/tasks/fuzz_task.py

```diff
@@ -96,46 +95,52 @@ class FuzzErrorCode(object):
 
 
 def get_unsymbolized_crash_stacktrace(stack_file_path):
-  try:
-    with open(stack_file_path, 'rb') as f:
-      return (utils.decode_to_unicode(f.read()), None)
-  except:
-    logs.log_error('Unable to read stacktrace from file %s.' % stack_file_path)
-    return (None, sys.exc_info()[1])
+  """Read unsymbolized crash stacktrace."""
+  with open(stack_file_path, 'rb') as f:
+    return (utils.decode_to_unicode(f.read()), None)
 
 
 class Crash(object):
   """Represents a crash (before creating a testcase)."""
 
-  def __init__(self, crash):
-    """Initialize
-
-    Args:
-      crash: An instance of fuzzing.testcase_manager.Crash.
-    """
-    self.file_path = crash.file_path
-    self.crash_time = crash.crash_time
-    self.return_code = crash.return_code
-    self.resource_list = crash.resource_list
-    self.gestures = crash.gestures
-    self.stack_file_path = crash.stack_file_path
+  @classmethod
+  def from_testcase_manager_crash(cls, crash):
+    """Create a Crash from a testcase_manager.Crash."""
+    try:
+      orig_unsymbolized_crash_stacktrace = (
+          get_unsymbolized_crash_stacktrace(crash.stack_file_path))
+    except Exception:
+      logs.log_error(
+          'Unable to read stacktrace from file %s.' % crash.stack_file_path)
```

Comment:
Does the actual exception get included in the log?

Reply:
Yes, log_error should include the exception in the log.